init_user_groups

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2023 Collabora Limited
+# Author: Jeny Sadadia <jeny.sadadia@collabora.com>
+
+"""Command line utility for creating default user groups"""
+
+import subprocess
+
+
+DEFAULT_GROUPS = ['admin', 'users']
```

Comment:
If users can belong to several groups, then I think we don't need the `users` group.

Reply:
As per our discussion yesterday, it makes sense to just have "admin" as a default group and discard "users" one. That way we can prevent all the regular users from accessing each other's data.